engine/src/main/java/org/destinationsol/common/SolMath.java

```diff
@@ -15,49 +15,72 @@
  */
 package org.destinationsol.common;
 
-import java.util.ArrayList;
-
-import org.destinationsol.Const;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import com.badlogic.gdx.math.MathUtils;
 import com.badlogic.gdx.math.Vector2;
 import com.badlogic.gdx.utils.JsonValue;
 import com.badlogic.gdx.utils.Pool;
+import org.destinationsol.Const;
+
+import java.util.ArrayList;
 
 /**
- * A class with helpful mathematical functions
+ * This class presents a collection of handy mathematical as well as programmatical functions, for working with angles,
+ * distances, speeds, vectors.
  */
 public class SolMath {
-    private static Logger logger = LoggerFactory.getLogger(SolMath.class);
+    /**
+     * The pi number constant.
```

Comment:
It puzzles me a bit why this class even wraps these `MathUtils` constants and such. Which came up on Slack too. Fine to add javadoc to them but it seems silly as opposed to just replacing all the actual calls to use `MathUtils` directly.

However, that's existing code, so I'm fine leaving it that way until somebody wants to take the time to throw all the silly stuff away, carefully :-)

Reply:
I'll agree that this extra layer of abstraction seems weird. Plus, the javadocs adds no real information either - everyone reading this code will know what `public static float PI` represents.

Either way, we can deal with this later.